src/layout/header/Header.tsx

```diff
@@ -0,0 +1,25 @@
+import './Header.styled.scss'
+
+import { useLocation, useNavigate } from 'react-router-dom'
+
+import { RoutesManager } from '../../routesManager'
+import { Logo } from '../../ui-base/logo/Logo'
+
+export const Header = () => {
+  const navigate = useNavigate()
+  const { pathname } = useLocation()
+
+  return (
+    <header className="container">
+      <Logo />
+      <div className="buttonsWrapper">
+        <button
+          className={pathname === '/home' ? 'active' : ''}
```

Comment:
**`Classnames`** utility will helps to avoid ternary operator 
```suggestion
          className={classNames({
            'active': pathname === '/home',
          })}
```

Reply:
Thank you! I hadn't used it before, but now I've tried it. Really useful utility. Done